### PR TITLE
Issue/none impl missing methods

### PR DESCRIPTION
**Pull request by wyrzyk**

### CODE REVIEW

File: src/test/java/com/atlassian/db/replica/it/DualConnectionIT.java [hunk 1]
```diff
@@ -97,6 +110,81 @@ public void shluldNotFailWhenChangingTransactionIsolationLevel() throws SQLExcep
         }
     }
 
+    @SuppressWarnings({"ThrowableNotThrown"})
+    @Test
+    public void shouldImplementAllConnectionMethods() throws SQLException {
```
Comment (dagguh) [inline src/test/java/com/atlassian/db/replica/it/DualConnectionIT.java:115]: Smoke test 👌 nice and easy

File: src/main/java/com/atlassian/db/replica/internal/NetworkTimeout.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        NetworkTimeout that = (NetworkTimeout) o;
+        return milliseconds == that.milliseconds && Objects.equals(executor, that.executor);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(executor, milliseconds);
```
Comment (dagguh) [inline src/main/java/com/atlassian/db/replica/internal/NetworkTimeout.java:31]: `executor.hashCode`? 🤔
Is `NetworkTimeout` held in hash collections?

Comment (wyrzyk) [reply src/main/java/com/atlassian/db/replica/internal/NetworkTimeout.java:31]: Why not? 😉

**Review by dagguh — APPROVED** [review]

File: src/test/java/com/atlassian/db/replica/it/DualConnectionIT.java [hunk 3]
```diff
@@ -97,6 +110,81 @@ public void shluldNotFailWhenChangingTransactionIsolationLevel() throws SQLExcep
...
+            connection.setClientInfo("ApplicationName", "app");
+            final Properties properties = new Properties();
+            properties.setProperty("ApplicationName", "app");
+            connection.setClientInfo(properties);
+            final int timeout = (int) Duration.ofSeconds(30).toMillis();
+            connection.setNetworkTimeout(Executors.newSingleThreadExecutor(), timeout);
+            assertThat(connection.getNetworkTimeout()).isEqualTo(timeout);
+            connection.abort(Executors.newSingleThreadExecutor());
+
+            connection.close();
```
Comment (afaruga-atlassian) [inline src/test/java/com/atlassian/db/replica/it/DualConnectionIT.java:184]: :hugefan

**Pull request merged**